app/src/main/java/de/danoeh/antennapod/fragment/ChaptersFragment.java

```diff
@@ -43,11 +45,26 @@ public class ChaptersFragment extends AppCompatDialogFragment {
     @NonNull
     @Override
     public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
-        return new MaterialAlertDialogBuilder(requireContext())
+
+        AlertDialog dialog = new MaterialAlertDialogBuilder(requireContext())
                 .setTitle(getString(R.string.chapters_label))
                 .setView(onCreateView(getLayoutInflater()))
-                .setNegativeButton(getString(R.string.close_label), null) //dismisses
-                .create();
+                .setPositiveButton(getString(R.string.close_label), null) //dismisses
+                .setNeutralButton("Reset", null)
```

Comment:
Please use a String resource that can be translated instead of hard-coding a String. I'm pretty sure we already have some `R.string` resource that says "Refresh".